Heroes.ReplayParser/Foole.Mpq/BitStream.cs

```diff
@@ -26,10 +26,11 @@
 // OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
 // WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 //
+
 using System;
 using System.IO;
 
-namespace Foole.Mpq
+namespace Heroes.ReplayParser.Foole.Mpq
```

Comment:
I'd rather not change the Foole.Mpq files, as they originally came from another package.  Could you revert the cosmetic changes to these Foole.Mpq files?

Reply:
Fixed with the latest commit